docs/usage/services.rst

```diff
@@ -66,17 +67,16 @@ Services provide high-level methods for common operations:
 
 .. code-block:: python
 
-    async def create_post_with_tags(
+    async def create_post(
         post_service: PostService,
         data: PostCreate,
     ) -> PostResponse:
         """Create a post with associated tags."""
-        # Service automatically validates input using PostCreate schema
```

Comment:
the service doesn't actually validate input, right? the data passed in to the service methods are assumed to be valid?

Reply:
Correct, this is a good update.